src/react-native/05-intro-to-testing/intro-to-testing.md

```diff
@@ -2,23 +2,84 @@
 @parent learn-react-native 5
 @outline 3
 
-@description TODO
+@description Learn about testing solutions in React Native.
 
 @body
 
 ## Overview
 
 In this section, you will:
 
-- TODO
+- Learn about React Native Testing Library
+- See the distinctions of testing for React Native components
 
 ## Objective: Testing in React Native
 
-TODO
+How do we know when our code is working correctly? How do we know it’s still working correctly in the future after we make changes to it?
 
-### Concept TODO
+Testing helps by verifying that given certain inputs our code generates expected outputs. So far we’ve copied existing tests to prove that we’ve completed the exercise correctly, now let’s dive in and learn about how React Native testing is done.
 
-TODO
+### Introducing React Native Testing Library
+
+[React Native Testing Library (RNTL)](https://callstack.github.io/react-native-testing-library/docs/getting-started) is a set of tools that helps us write robust tests for our React Native applications. Unlike some other testing libraries that focus on the internal state and implementation details of components, RNTL emphasizes testing the behavior of components as experienced by the end-users.
+In order to test components it provides light utility functions using React Test Renderer; most testing libraries use the DOM, but since React Native doesn't use the DOM, the React Test Renderer is necessary.
```

Comment:
`components it` -> `components, it`

Also, replace the semicolon with a period.